yaksh/models.py

```diff
@@ -293,7 +293,7 @@ class Profile(models.Model):
         choices=[(tz, tz) for tz in pytz.common_timezones]
     )
     is_email_verified = models.BooleanField(default=False)
-    activation_key = models.CharField(max_length=40, blank=True, null=True)
+    activation_key = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
is_email_verified field cannot have default=True since in production it will not send email, because in views the condition is there which will check this variable and it will find this variable to be True and will not send mail